lib/matplotlib/colorizer.py

```diff
@@ -0,0 +1,483 @@
+"""
+The Colorizer class which handles the data to color pipeline via a
+normalization and a colormap.
+
+.. seealso::
+
+  :doc:`/gallery/color/colormap_reference` for a list of builtin colormaps.
+
+  :ref:`colormap-manipulation` for examples of how to make
+  colormaps.
+
+  :ref:`colormaps` an in-depth discussion of choosing
+  colormaps.
+
+  :ref:`colormapnorms` for more details about data normalization.
+"""
+
+import numpy as np
+from numpy import ma
+import functools
+from matplotlib import _api, colors, cbook, scale, cm
+
+
+class Colorizer():
+    """
+    Class that holds the data to color pipeline
+    accessible via `.to_rgba(A)` and executed via
```

Comment:
Seeing as this is the main operation of the `Colorizer` class, I changed the the name of this function from `to_rgba()` to a simple `__call__`. 
@timhoffm do you agree that `Colorizer.__call__` is the best call signature for the norm→color pipeline?

Reply:
I'm undecided. My general point of view is that there are only very few cases where `__call__` is justified (mostly drop-in replacements for callables). It's generally better to have an explicit name for the action; otherwise, you end up with stuff like `plt.get_cmap('jet')(0.5)`. OTOH norms and cmaps work that way, so it's not completely unreasonable to keep that pattern for `Colorizer`.

Reply:
I changed it back to to_rgba, se Toms comment below.